client/src/components/_common/InfiniteScrollTable/InfiniteScrollTable.scss

```diff
@@ -36,42 +36,34 @@
   }
 
   tbody {
-    tr:not([class=-status]) td {
+    tr:not(.-status) td {
       border-bottom: 1px solid rgba(0, 0, 0, 0.1);
     }
-
-    tr:nth-child(even):not([class=-status]) {
+    tr:nth-child(even):not(.-status) {
       background-color: rgba(0, 0, 0, 0.03);
     }
-    tr:hover:not([class=-status]) {
+    tr:hover:not(.-status) {
       background-color: rgba(0, 0, 0, 0.05);
     }
 
-    /* !!!: Temporary lint disabling to make simpler PR diff */
-    /* stylelint-disable no-descending-specificity */
-    :not(.-status) {
-    td {
+    tr:not(.-status) td {
       vertical-align: middle;
       padding: 0.7em var(--cell-horizontal-padding);
       margin: auto 0; /* vertically center span content */
       overflow: hidden;
       text-overflow: ellipsis;
       white-space: nowrap;
     }
-    }
 
-    .-status {
+    tr.-status {
```

Comment:
This change offers consistency and fixes a lint problem that is difficult to diagnose the first time. I was familiar with it. I don't suggest this over-specificity (just `.-status` is adequate), but I'm fixing lint issues, not refactoring, in this PR.